### PR TITLE
Top bar updates

### PR DESCRIPTION
**Pull request by kaytcampbell**
https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/13865

Styles moneris iframe inputs, adds mobile styling for non-logged in nav, makes logo clickable.

image
image

### PR Review Checklist
To be completed by the person who opens the PR. Use strikethroughs for items that are not applicable in list.
- [ ] I have reviewed my own PR to check syntax & logic, removed unnecessary/old code, made sure everything aligns with our style guide and BEM.
- [ ] I've added new components to the style guide (or have a PR in to add them).
- [ ] Any applicable version numbers have been updated.
- [ ] I've tested on mobile, tablet, and desktop, as well as across all of the browsers we support.
- [ ] I've proof-read all text for legibility, proper grammar, punctuation, and capitalization (titlecase).
- [ ] I have written a detailed PR message explaining what is being changed and why
- [ ] I’ve linked to any relevant VSTS tickets
- [ ] I’ve added the appropriate review symbols to my PR - (elephant) for dev review, (art) for design

### CODE REVIEW

File: spec/ama_layout/decorators/moneris_decorator_spec.rb [hunk 1]
```diff
@@ -36,16 +36,19 @@
...
       expect(moneris_presenter.textbox).to include "display: block;"
-      expect(moneris_presenter.textbox).to include "font-family: 'Helvetica Neue', Helvetica, Roboto, Arial, sans-serif;font-size: 14px;"
-      expect(moneris_presenter.textbox).to include "font-style: normal;font-variant: normal;"
+      expect(moneris_presenter.textbox).to include "font-family: 'Open Sans', Helvetica, Roboto, Arial, sans-serif;"
+      expect(moneris_presenter.textbox).to include "font-size: 18px;"
+      expect(moneris_presenter.textbox).to include "font-style: normal;"
+      expect(moneris_presenter.textbox).to include "font-variant: normal;"
       expect(moneris_presenter.textbox).to include "font-weight: normal;"
-      expect(moneris_presenter.textbox).to include "height: 44px;"
+      expect(moneris_presenter.textbox).to include "height: 63px !important;"
```
Comment (ryan-reddekopp) [inline spec/ama_layout/decorators/moneris_decorator_spec.rb:44]: Is the !important necessary?

**Comment by ryan-reddekopp** [comment]
Screen Shot 2020-08-17 at 9 22 18 AM
I noticed some overlap with the sign out notification and the header in mobile. Is that something that needs to be fixed in ama_layout?

File: spec/ama_layout/decorators/moneris_decorator_spec.rb (hunk 1 above)
Comment (kaytcampbell) [reply spec/ama_layout/decorators/moneris_decorator_spec.rb:44]: Yes, there is a height being declared inline on all the fields, so until we can remove that height the important is necessary to override it.

Comment (ryan-reddekopp) [reply spec/ama_layout/decorators/moneris_decorator_spec.rb:44]: That sounds great!

**Pull request merged**